airbyte-integrations/connectors/source-zuora/source_zuora/zuora_errors.py

```diff
@@ -0,0 +1,79 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+
+import sys
+
+import requests
+from airbyte_cdk import AirbyteLogger
+
+
+class Error(Exception):
+    """ Base Error class for other exceptions """
+
+    # Define the instance of the Native Airbyte Logger
+    logger = AirbyteLogger()
+
+
+class ZOQLQueryError(Error):
+    """ Base class for  ZOQL EXPORT query errors """
+
+    def __init__(self, response: requests.Response = None):
+        if response:
+            self.response = response.json()
+            self.error_msg = self.response["data"]["errorMessage"]
+            self.query = self.response["data"]["query"]
+            super().__init__(self.logger.error(f"{self.error_msg}, QUERY: {self.query}"))
```

Comment:
where do these instance variables else use? I propose to use their local variants.